src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+import { IDrop } from "../../types/dropdown";
+import {
+  DropdownBtn,
+  DropdownContainer,
+  DropdownList,
+  DropdownMenu,
+} from "./DropdownStyle";
+
+export const Dropdown = ({ items, dropdownValue, setDropdownValue }: IDrop) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <DropdownContainer>
+      <DropdownBtn onClick={() => setIsOpen(!isOpen)}>
+        {
+          items.find(({ value }) => value === dropdownValue)
+            .name
+        }
+      </DropdownBtn>
+      <DropdownMenu>
+        {items.map(
+          ({
+            id,
+            name,
+            value,
+          }) => (
+            <DropdownList
+              onClick={() => {
+                setDropdownValue(value);
```

Comment:
if click action has more than 2 actions - better to create handleClick function
(changes are not needed, just hint)